Loop/Managers/DeviceDataManager.swift

```diff
@@ -587,13 +587,18 @@ final class DeviceDataManager {
                         "* profileExpiration: \(Bundle.main.profileExpirationString)",
                         "* gitRevision: \(Bundle.main.gitRevision ?? "N/A")",
                         "* gitBranch: \(Bundle.main.gitBranch ?? "N/A")",
+                        "* isWorkspaceBuild: \(Bundle.main.workspaceBuild ?? "No")",
+                        "* gitRevWorkspace: \(Bundle.main.gitRevWs ?? "N/A")",
+                        "* gitBranchWorkspace: \(Bundle.main.gitBranchWs ?? "N/A")",
                         "* sourceRoot: \(Bundle.main.sourceRoot ?? "N/A")",
                         "* buildDateString: \(Bundle.main.buildDateString ?? "N/A")",
                         "* xcodeVersion: \(Bundle.main.xcodeVersion ?? "N/A")",
                         "",
                         "## FeatureFlags",
                         "\(FeatureFlags)",
                         "",
+                        alertReport,
```

Comment:
Just a couple notes about ordering: I assume this is how *you* would sort it because you find this more useful than the other fields. As I generally have to look into algorithm issues, the dosing info (carbs/doses) are more important for me. I'll take this change as the file is already so unwieldy already we generally need to navigate by search anyways. But please don't look at this a file to optimize and spend time on for general users. If there are things in here that users need on a regular basis, we should be spending time figuring out how to expose them in the UI.

Reply:
I was thinking of the use case where there was a pod failure and the user wanted to see the alert on their phone.
It is very painful to scroll long distances in a file on the phone interface and this would make it easier to find.  
Note - I did this before you added the UI that showed it for last pod.
Even with that feature, there may be cases of back-to-back faults and user could see both in the report.

Reply:
Thanks for the changes. Tracking LoopWorkspace git revision/branch will probably be useful to those who need to dig into Loop issue reports, and the shorter sha hash is nice.